08-error-handling.js

```
@@ -25,49 +25,49 @@ try {
     return data.toUpperCase();
 }
 
 // TODO: Creating custom errors
-class ValidationError extends /* PLACEHOLDER */ {
+class ValidationError extends Error {
     constructor(message, field) {
-        /* PLACEHOLDER */(message);
+        throw new Error(message);
         this.name = 'ValidationError';
-        this.field = /* PLACEHOLDER */;
+        this.field= field;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)